lib/larch/base-backend.js

```diff
@@ -59,11 +59,15 @@ function BaseBackend(options) {
 BaseBackend.prototype.log = function log(record, cb) {};
 
 BaseBackend.prototype.bootstrap = function bootstrap(cb) {
-    cb();
+    if (typeof cb === 'function') {
```

Comment:
please don't introduce optional callback pattern, inject noop if you don't care.